README.md

```diff
@@ -147,13 +147,27 @@ These instructions will get you a copy of the project up and running on your loc
 - `make`
 - `git`
 
-## Getting Started
+## Using `make`
 
 We've made the startup process simple by using a Makefile for all common workflows.
 
-To see what Makefile commands are available, in your terminal shell run `make` or `make help`. It will list all the available Makefile commands with their descriptions, such as
+To see what Makefile commands are available, in your terminal shell run `make` or `make help`. It will list all the available Makefile 
+commands with their descriptions.
 
-```Makefile
-install                        Install client and server side packages for development
-start                          Start the development server
-```
+| Target    | Effect                                                  |
+|-----------|---------------------------------------------------------|
+| `help`    | Display all other make targets and their effects        |
+| `install` | Install client and server side packages for development |
+| `start`   | Start the development server                            |
+
+## Using `yarn` or `npm`
```

Comment:
Is there a benefit from promoting multiple development tools here? (make vs npm)

IMO we should push one officially supported method, my opinions on which one are more loosely held.

Reply:
In my experience, creating `npm` scripts leads to a more enjoyable and simpler development process. `npm` a prerequisite for installing dependencies, building, and running most node projects, while `make` adds a signifigant system constraint without adding much value in this domain. For these reasons, I believe we should always create cononical `npm` scripts and create `make` targets that alias said scripts.

Reply:
Valid point, but at Ticket Evolution we have more than just node projects. I'm thinking we add all required workflows for every project (build for prod, test, run in dev mode, install, etc) to Make, and anything optional (running analytics, coverage, w/e) we leave out of the Makefile. That way everyone can clone, install, and get each app running, and if you want to venture into any advanced workflow you can use the underlying build tools directly.

Reply:
I really like that idea!

Reply:
So, should we document both?

Reply:
Yeah leave this as is, I just talked myself into adding documentation for both ;)

As long as we all treat the make commands as a public API, they should always work as expected